cpp/src/io/parquet/page_delta_decode.cu

```diff
@@ -380,35 +381,35 @@ CUDF_KERNEL void __launch_bounds__(decode_delta_binary_block_size)
     uint32_t target_pos;
     uint32_t const src_pos = s->src_pos;
 
-    if (t < 2 * warp_size) {  // warp0..1
+    if (warp.meta_group_rank() == 0 or warp.meta_group_size() == 1) {  // warp0..1
```

Comment:
Are these supposed to use the same function? One is `meta_group_rank` and one is `meta_group_size`. I'm guessing you meant this:
```suggestion
    if (warp.meta_group_rank() == 0 or warp.meta_group_rank() == 1) {  // warp0..1
```

Reply:
Nice catch!